lib/MusicBrainz/Server/Constants.pm

```diff
@@ -6,6 +6,7 @@ use warnings;
 use base 'Exporter';
 
 use DateTime::Duration;
+use DateTime::Locale '1.00';
```

Comment:
Just to make sure: this means _at least version 1.00_, right?

Reply:
That is correct, see https://perldoc.perl.org/functions/use.html